nucleus/deploy/nucleus_integration.py

```diff
@@ -0,0 +1,156 @@
+# This file contains all the Core Nucleus <-> HMI integrations
```

Comment:
@sasha-scale @ardila would be good to get your 👀 on this. Bear in mind this is temporary - the orchestration code in here would likely be moved to some Nucleus API endpoint (as discussed today, regarding https://app.shortcut.com/scaleai/story/350521/amend-model-creation-api-endpoint-to-optionally-be-associated-with-a-model-bundle-id-if-it-exists-write-to-nucleus-model)

Reply:
cc @Tianwei-She

Reply:
Hmm this stuff does add the model upload + bundle storing in the db in one go, might wanna do something similar in the Nucleus API endpoint we talked about